pep-0xxx.rst

```diff
@@ -888,35 +888,169 @@ limitations of the above implementations. Specifically, as every implementation
 *except* OpenSSL requires that each individual cipher be provided, there is no
 option but to provide that lowest-common denominator approach.
 
-The simplest approach is to provide an enumerated type that includes all of the
-cipher suites defined for TLS. The values of the enum members will be their
-two-octet cipher identifier as used in the TLS handshake, stored as a tuple of
-integers. The names of the enum members will be their IANA-registered cipher
-suite names.
-
-Rather than populate this enum by hand, it is likely that we'll define a
-script that can build it from Christian Heimes' `tlsdb JSON file`_ (warning:
-large file). This also opens up the possibility of extending the API with
-additional querying function, such as determining which TLS versions support
-which ciphers, if that functionality is found to be useful or necessary.
+The simplest approach is to provide an enumerated type that includes a large
+subset of the cipher suites defined for TLS. The values of the enum members
+will be their two-octet cipher identifier as used in the TLS handshake,
+stored as a 32 bit integer. The names of the enum members will be their
```

Comment:
Heh, only just noticed this, but: two octets is definitely a 16-bit integer, not a 32-bit one. ;)